src/components/ThumbnailNavigation.js

```diff
@@ -52,12 +53,22 @@ export class ThumbnailNavigation extends Component {
    */
   cellRenderer(options) {
     const {
-      columnIndex, key, style,
+      columnIndex, key, style, rowIndex,
     } = options;
     const {
-      classes, window, setCanvas, config, canvasGroupings, t,
+      classes, window, setCanvas, config, canvasGroupings, t, position,
     } = this.props;
-    const currentGroupings = canvasGroupings.groupings()[columnIndex];
+    let currentIndex = null;
+    switch (position) {
```

Comment:
Could do this with a ternary, since I think there will only ever be 2 options.

`currentIndex = (position === 'right') ? rowIndex : columnIndex`

Reply:
Sure thing. I've made the requested change.